pubspec.yaml

```diff
@@ -10,6 +10,7 @@ dependencies:
   checked_yaml: ^2.0.0
   codemirror: ^0.7.6+5.65.5
   collection: ^1.16.0
+  encrypt: ^5.0.1
```

Comment:
I don't know much about this package, but I see it wraps the [pointycastle](https://pub.dev/packages/pointycastle)  package. Why not use pointycastle directly?

Reply:
The encrypt package provides a higher level interface to pointycastle and saves having to write the wrapper code ourselves.   I did a thorough review of the encrypt package before using it, and i did submit a minor PR or two, but overall found it to be simple and well engineered. 
In looking at pub.dev it appears that pointycastle is more often used through the encrypt package then directly, probably because ease of use versus the lower level nature of pointycastle.

Reply:
Okay, the reason I ask is that this can come up in security reviews in the future so it's important to have a strong justification for the extra code.